Codeinwp/neve
# Feat/performance module hooks

### Summary
Adds support functionality for the performance module.
References:  https://github.com/Codeinwp/neve-pro-addon/issues/1271

### Will affect visual aspect of the product
NO

### Test instructions
It should not change the default behavior of the theme.
It should add additional classes to the theme if the performance module lazy-load content is being used.
